query-node/mappings/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "mappings",
+  "version": "0.0.1",
+  "description": "Mappings for hydra-processor",
+  "main": "lib/mappings/index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "rm -rf lib && tsc --build tsconfig.json",
+    "lint": "echo \"Skippinng\"",
+    "clean": "rm -rf lib"
+  },
+  "dependencies": {
+    "@dzlzv/hydra-common": "0.0.3",
+    "@polkadot/types": "~2.10.2-7"
```

Comment:
Has this version of @polkadot/types been tested against the babylon runtime?
The monorepo yarn workspace will anyway override this to `v1.26.1`. Will that be a problem ?

Reply:
New mappings will depend on this version because of the typegen stuff but not sure how we can manage that.

Reply:
We will have to test but I'm concerned the newer version will break the other apps. In that case we may need to move query-node out of the top level yarn workspace.